### PR TITLE
add marketplace instructions

### PR DESCRIPTION
**Pull request by davidmyriel**
This PR adds a rudimentary set of instructions for AWS users:
1. How to use AWS marketplace.
2. Billing Details
3. Considerations
4. Further Cloud Instructions

### CODE REVIEW

File: qdrant-landing/content/documentation/cloud/aws-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Using this option is simple and isn't any more different than directly signing up with Qdrant Cloud. 
+
+## Billing
+
+You might see that AWS gives an ambiguous price for Qdrant. It is common to abstract usage based pricing in the AWS marketplace, as there are too many factors to model billing from the AWS side. 
+
+![pricing](/docs/cloud/pricing.png)
+
+The pricing you'll see in Qdrant Cloud while booking will apply normally.
+However, the payment carried out via your AWS account. To get a clearer idea for the pricing structure, please use our [Billing Calculator](https://cloud.qdrant.io/calculator).
```
Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/aws-marketplace.md:20]:
```suggestion
However, the payment is carried out via your AWS account. To get a clearer idea for the pricing structure, please use our [Billing Calculator](https://cloud.qdrant.io/calculator).
```

File: qdrant-landing/content/documentation/cloud/aws-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: AWS Marketplace
+weight: 60
+---
+
+# Qdrant Cloud on AWS Marketplace
+
+## Overview
+
+Our [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-rtphb42tydtzg) listing makes it simpler to access Qdrant for users who wish to host all of their infrastructure on Amazon Web Services.
```
Comment (fabriziobonavita) [inline qdrant-landing/content/documentation/cloud/aws-marketplace.md:10]: Can we make sure customers understand here that the clusters run on AWS, but within our infrastructure?

**Review by fabriziobonavita — COMMENTED** [review]
Great PR - Thank you!

File: qdrant-landing/content/documentation/cloud/aws-marketplace.md (hunk 2 above)
Comment (pansen) [reply qdrant-landing/content/documentation/cloud/aws-marketplace.md:10]: It is not about hosting, but about *billing*.

**Pull request merged**